docs/api/api_blueprint.apib

```diff
@@ -14868,6 +14871,7 @@ Update a single attendee by `id`.
                     "country": "IN",
                     "is-checked-in": "true",
                     "checkin-times": "2017-12-13T22:59:59.123456+00:00",
+                    "attendee-notes": "example",
```

Comment:
This should have been inserted six times atleast in the file. I only see it inserted three times. Github isn't allowing me to open the whole file as it is too long.
Please check that you've added it in the below ones: 
1. POST - in both request and response
2. GET - in response
3. PATCH - in both request and response
4. GET a list of attendees of an order - in the response.

Reply:
@srv-twry Added in all of them.